packages/dht/src/dht/DhtNode.ts

```diff
@@ -46,15 +46,15 @@ import { isNodeJS } from '../helpers/browser/isNodeJS'
 import { sample } from 'lodash'
 
 export interface DhtNodeEvents {
-    newContact: (peerDescriptor: PeerDescriptor, closestPeers: PeerDescriptor[]) => void
-    contactRemoved: (peerDescriptor: PeerDescriptor, closestPeers: PeerDescriptor[]) => void
+    newContact: (contact: DhtPeer, closestContacts: DhtPeer[]) => void
```

Comment:
Better to emit just the PeerDescriptor here , or something more abstract like `Remote`

Reply:
Reverted to use `PeerDescriptor`